package.json

```diff
@@ -15,7 +15,9 @@
 	],
 	"scripts": {
 		"build": "npx nx run-many --target=build --all --parallel",
+		"build:production": "npm run build && npm run _build -w @inlang/website && npm run _build -w @inlang/editor",
```

Comment:
I would remove this. If you put @inlang/website and @inlang/editor into the dependencies, monorepo tools will automatically build the required dependencies. 

I would also remove the `_` from the build scripts from editor and website, we should build everything as required. If things become too slow, we should re-enable nx cloud

Reply:
Done ✅